source/_meta/_01-foot.twig

```diff
@@ -1,5 +1,8 @@
 {{ patternLabFoot|raw }}
 
+{# Add one for each js file to load #}
+{#<script src="[PATH-TO-JS]"></script>#}
```

Comment:
Can already use `<script defer src="../../js/dist/main.bundle.js?{{ cacheBuster }}"/>` in the `<head>`?

Reply:
Along with `vendors~main.bundle.js`, yes. I think the naming is predictable now.

Reply:
Although probably wouldn't want to `defer` as `DOMContentLoaded` is blocked from firing until all deferred scripts are loaded.